modules/homefinder/widgets/homefinder-widgets.js

```diff
@@ -0,0 +1,33 @@
+(function () {
+	hf_data.forEach(function(data, index) {
```

Comment:
__[jshint]__
* 'hf_data' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->